From 3548c5cbbe48e51213de839ea43ba37b986c37d6 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:33 -0400
Subject: [redhat] kabi: Adding symbol bio_endio (fs/bio.c)

Message-id: <97de23ca062edd943445881b3114d2c7ce274fc0.1410510773.git.poros@redhat.com>
Patchwork-id: 95924
O-Subject: [PATCH RHEL7.1 033/233] kabi: Adding symbol bio_endio (fs/bio.c)
Bugzilla: 1062361 1071097
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    End I/O on a bio.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/bio_endio
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/bio_endio
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/bio_endio

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/bio_endio  b/redhat/kabi/kabi-module/kabi_ppc64/bio_endio 
new file mode 100644
index 0000000..9a4f1f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/bio_endio  
@@ -0,0 +1,2 @@
+#1-
+0xfe7394c3 bio_endio vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/bio_endio  b/redhat/kabi/kabi-module/kabi_s390x/bio_endio 
new file mode 100644
index 0000000..96f650e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/bio_endio  
@@ -0,0 +1,2 @@
+#1-
+0x8c143b05 bio_endio vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/bio_endio  b/redhat/kabi/kabi-module/kabi_x86_64/bio_endio 
new file mode 100644
index 0000000..dc4a9b3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/bio_endio  
@@ -0,0 +1,2 @@
+#1-
+0xdd5fbc2b bio_endio vmlinux EXPORT_SYMBOL
-- 
1.7.1